CONTRIBUTING.md

```diff
@@ -0,0 +1,124 @@
+# Contributing
```

Comment:
If you move this to under the github folder then I think github knows to use it for the contributing tab?

Reply:
From https://blog.github.com/2012-09-17-contributing-guidelines/

It should be project root, where it currently lives